README.md

```diff
@@ -7,12 +7,12 @@ Sample senders and receivers to be used with the Switchboard project. https://gi
 
 # Usage
 1. Create your virtual environment and install the required packages  
-`python3 -m venv env`  
+`python -m venv env`  
 `source env/bin/activate` (or `env\Scripts\activate.bat` for Windows users)  
 `pip install -r requirements.txt`  
 
 2. Run the video receiver  
-`python3 receiver.py --ip <ip>`
+`python receiver-ui.py`
 
 3. Run the video sender  
-`python3 sender.py --file <file> --ip <ip>`
+`python sender-ui.py`
```

Comment:
Now that the code is in the `src` folder, these commands don't work from the top-level project folder.
Either we should add `src/` to the path of the files e.g. `python src/sender-ui.py` or indicate that you need to `cd` into the src folder. However `python src/sender-ui.py` doesn't work because it crashes when loading the image; see my later comment about that.

Reply:
Addressed in earlier comment